*** /tmp/oauthtwitter.py	2010-10-10 21:52:54.667007066 +0100
--- /usr/lib/python2.6/site-packages/oauthtwitter.py	2010-10-10 22:05:23.296343192 +0100
***************
*** 79,85 ****
  
          if http_method == "POST":
              encoded_post_data = req.to_postdata()
!             url = req.get_normalized_http_url()
          else:
              url = req.to_url()
              encoded_post_data = ""
--- 79,85 ----
  
          if http_method == "POST":
              encoded_post_data = req.to_postdata()
!             url = req.normalized_url
          else:
              url = req.to_url()
              encoded_post_data = ""
